src/main/webapp/app/view/project/project.routes.ts

```diff
@@ -0,0 +1,35 @@
+import { Routes } from '@angular/router';
+
+import { ProjectResolve } from 'app/entities/project/project.route';
+import { UserRouteRoleAccessService } from 'app/core/auth/user-route-role-access-service';
+
+import { ProjectComponent } from './project.component';
+import { ScreenplayComponent } from 'app/view/project/screenplay/screenplay.component';
+
+import { ROLE_ADMIN, ROLE_SECRETARY } from 'app/shared/constants/role.constants';
+
+export const PROJECT_ROUTES: Routes = [
```

Comment:
Dasselbe hier wie oben. Es gibt 2 files die genau gleich heissen. Wäre sehr stark dafür, Code, den wir nicht brauchen, zu entfernen.

Reply:
sind aber nicht im gleichen Ordner : )
der bestehende Code wir ja (fast bis gar) nie angefasst werden. er bietet aber dem Applikations `ADMIN` eine gute Übersicht. im Szenario, dass der TBO z.B. einmal eine Instanz verwalten wird, nicht wir. Die müssen dann eventuell Benutzern helfen und können kein SQL.

Reply:
conversation in #54.